modules/apps/object/object-service/src/main/java/com/liferay/object/internal/notification/term/contributor/ObjectDefinitionNotificationTermEvaluator.java

```diff
@@ -199,25 +199,29 @@ private String _evaluateParentObjectDefinitionAuthor(
 				continue;
 			}
 
-			User user = null;
-
 			ObjectField objectField = _objectFieldLocalService.getObjectField(
 				objectRelationship.getObjectFieldId2());
 
+			long primaryKey = GetterUtil.getLong(
+				termValues.get(objectField.getName()));
+
+			if (Objects.isNull(primaryKey)) {
+				return StringPool.BLANK;
+			}
+
+			User user = null;
+
 			if (objectDefinition.isSystem()) {
 				user = _userLocalService.getUser(
 					MapUtil.getLong(
 						_objectEntryLocalService.getSystemModelAttributes(
-							objectDefinition,
-							GetterUtil.getLong(
-								termValues.get(objectField.getName()))),
+							objectDefinition, primaryKey),
 						"creator"));
 			}
 			else {
 				ObjectEntry objectEntry =
 					_objectEntryLocalService.getObjectEntry(
-						GetterUtil.getLong(
-							termValues.get(objectField.getName())));
+						GetterUtil.getLong(primaryKey));
```

Comment:
I don't think we need to use `GetterUtil.getLong()` function because you already use it in line 205-206. What do you think?